arm9/lib/include/OS_system.h

```diff
@@ -17,17 +17,22 @@ typedef enum {
     OS_PROCMODE_SYS=31
 } OSProcMode;
 
+#pragma enumsalwaysint on
```

Comment:
Replace with macros defined in include-mw (so that we can replace in the case of a gnu build)